src/SFA.DAS.QnA.Database/projects/roatp/sections/section1.4.json

```diff
@@ -388,7 +388,7 @@
                 },
                 {
                     "Action": "NextPage",
-                    "ReturnId": "200",
+                    "ReturnId": "230",
```

Comment:
This page is for Main/Supporting providers, there's an equivalent page (161) for Employer providers, the equivalent change will need to be applied to that page too

Reply:
The routing for all 3 Provider types - Main, Employer and Supporting  is fixed now